pages/job-category/[JobCategory].js

```diff
@@ -0,0 +1,18 @@
+import { styled } from "styled-components";
```

Comment:
We don't need to de-structure here. [See notes for example](https://github.com/neuefische/muc-web-23-4/blob/main/sessions/react-styled-components/react-styled-components.md#styling-a-custom-component). 

```suggestion
import  styled from "styled-components";
```